### ISSUE TITLE
Make the "disable" directive ignore URLs.

### ISSUE THREAD

**Issue by matthewpersico** [koalaman/shellcheck#2506]
#### For bugs
- Rule Id (if any, e.g. SC1000): 
- My shellcheck version (`shellcheck --version` or 'online'): online
- [x] I tried on shellcheck.net and verified that this is still a problem on the latest commit
- [ ] It's not reproducible on shellcheck.net, but I think that's because it's an OS, configuration or encoding issue

#### For new checks and feature suggestions
- [x] shellcheck.net (i.e. the latest commit) currently gives no useful warnings about this
- [X] I searched through https://github.com/koalaman/shellcheck/issues and didn't find anything related

#### Here's a snippet or screenshot that shows the problem:

```sh
#!/bash

# shellcheck disable=SC2086 #https://github.com/koalaman/shellcheck/wiki/SC2086
```

#### Here's what shellcheck currently says:
```

```
#### Here's what I wanted or expected to see:

This is how I document my `disable` directives so that anyone in any decent editor can go directly to the Wiki page and see what I am talking about. It would be nice if the wiki page had a shortened url alias and the `disable` directive knew how to ignore it so that I could type:

```
#!/bash

# shellcheck disable=https://bitlysomething/SC2086
```

**Comment by matthewpersico** [comment]
I would settle for 
```
#!/bash

# shellcheck disable=https://github.com/koalaman/shellcheck/wiki/SC2086
```
